internal/service/resourcepolicy/resource_schema.go

```diff
@@ -0,0 +1,124 @@
+package resourcepolicy
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func ResourceSchema(ctx context.Context) schema.Schema {
+	return schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"created_by_user": schema.SingleNestedAttribute{
+				Description: "The user that last updated the atlas resource policy.",
+				Computed:    true,
+				Attributes: map[string]schema.Attribute{
+					"id": schema.StringAttribute{
+						Description: "Unique 24-hexadecimal character string that identifies a user.",
+						Computed:    true,
+					},
+					"name": schema.StringAttribute{
+						Description: "Human-readable label that describes a user.",
```

Comment:
no MarkdownDescription ?

Reply:
Added in [4fe132e](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2548/commits/4fe132e71bd9d3e29b54d3ac202349d6b05a0cfa).
Hopefully, one day we only need one `Description` 😅

Reply:
ha ha, i have a PR in my mind ;-)